src/layout/Header/index.tsx

```diff
@@ -13,60 +14,95 @@ const Header = () => {
-              onMouseLeave={handleMouseLeave}
-            >
-              <a
-                title="Games"
-                href="#"
-                className={[styles.nav__list_item, styles['nav__list_item--inactive']].join(' ')}
-              >
-                Games
-                <img src="/down_arrow.svg" alt="down array"></img>
-              </a>
-              {isHovered && (
-                <div className=" absolute top-[46px] w-[215px] rounded-[14px] bg-white shadow-[0_343px_343px_0_rgba(23,64,51,0.08)] ">
-                  <DropDownMenu menuContent={['Sprint →', 'Audio-call →']} />
-                </div>
-              )}
+    <div>
+      <header className="flex h-[88px] items-center justify-center bg-tk_greyLight shadow-xl">
+        <div className="flex w-full items-center justify-between px-[8px] md:px-[32px] lg:w-[1180px] lg:px-0">
+          <nav className="flex items-center gap-[30px]">
+            <h1 className="font-Dela_Gothic_One text-[21px] font-normal tracking-[-0.04em]">
+              TaoTalk
+            </h1>
+            <div className="hidden md:flex md:gap-[30px]">
+              <span className="text-tk_greyMedium">|</span>
+              <ul className="flex list-none items-center pl-0">
+                <li>
+                  <a
+                    title="Home"
+                    href="#"
+                    className="flex pl-0 pr-[30px] font-Montserrat leading-[1.4em] tracking-[-0.04em] text-black no-underline"
+                  >
+                    Home
+                  </a>
+                </li>
+                <li>
+                  <a
+                    title="Textbook"
+                    href="#"
+                    className="flex pl-0 pr-[30px] font-Montserrat leading-[1.4em] tracking-[-0.04em] text-tk_greyDark no-underline"
+                  >
+                    Textbook
+                  </a>
+                </li>
+                <li>
+                  <a
+                    title="Statistics"
+                    href="#"
+                    className="flex pl-0 pr-[30px] font-Montserrat leading-[1.4em] tracking-[-0.04em] text-tk_greyDark no-underline"
+                  >
+                    Statistics
+                  </a>
+                </li>
+                <li
+                  className="relative"
+                  onMouseEnter={handleMouseEnter}
+                  onMouseLeave={handleMouseLeave}
+                >
+                  <a
+                    title="Games"
+                    href="#"
+                    className="flex pl-0 pr-[30px] font-Montserrat leading-[1.4em] tracking-[-0.04em] text-tk_greyDark no-underline"
+                  >
+                    Games
+                    <img src="/down_arrow.svg" alt="down array"></img>
+                  </a>
+                  {isHovered && (
+                    <div className=" absolute top-[46px] w-[215px] rounded-[14px] bg-white shadow-[0_343px_343px_0_rgba(23,64,51,0.08)] ">
```

Comment:
https://github.com/taodemy/taotalk/blob/d8f609ef26d9a3dc290c100b53baff68a5dfc43a/src/layout/Header/index.tsx#L56

1. The box shadow setting s in figma is a combination of 6 shadows:
box-shadow: 0px 11.449064254760742px 11.449064254760742px 0px #3429270A;
box-shadow: 0px 11.449064254760742px 68.69439697265625px 0px #3429270A;
box-shadow: 0px 68.69439697265625px 68.69439697265625px 0px #3429270A;
box-shadow: 0px 91.59251403808594px 91.59251403808594px 0px #3429270A;
box-shadow: 0px 183.18502807617188px 183.18502807617188px 0px #3429271F;
box-shadow: 0px 343.4719543457031px 343.4719543457031px 0px #34292714;
I think there might be some problem here. So I chose the first box-shadow and set it in tailwind. The final visual effect is quiet close.  I think it could work.

2. As for the dropdown, I fould if I set the position method to "relative", the dropdown will not disappear. However, if the dropdown are positioned as "relative", when it pops up, it may disturb the original page layout and cause other problems. And I found cases online tend to use "absolute" positioning for dropdown.
So I stick on to using dropdown in "absolute" positioning way, and move the dropdown closer to the button link. Then when the mouse moves, the dropdown will not dissappear.